src/common/cvt.cpp

```diff
@@ -136,6 +136,76 @@ const SINT64 INT64_LIMIT = ((((SINT64) 1) << 62) / 5);
 // AP,2012: Looks like there is no need making len zero, but I keep old define for a reference.
 // {if (len) {memcpy(to, from, len); from += len; to += len; len = 0;} }
 
+#define DEFINE_ENUM_BITWISE_OPERATORS(enumType) \
```

Comment:
I would move this defines to some generic header as it could be useful for others `enum class` code and `cvt.cpp` already has many things.

Reply:
Done